src/OpenSage.Game/Logic/Object/Colliders.cs

```diff
@@ -15,6 +15,8 @@ protected Collider(Transform transform)
             Transform = transform;
         }
 
+        public abstract float GetHeight();
```

Comment:
Shouldn't this be an abstract property instead? `public abstract float Height { get; }`